python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
Yes but way beyond the scope of this PR which was trying to address common bottlenecks which are showing up.
Fixing the classes which are exploiting this whilst being able to load older jobs is potentially extremely difficult.
I raised this a while ago but I don't think there is an issue.